bootstrap/secret/secure.go

```diff
@@ -45,7 +45,8 @@ const (
 	TokenTypeKeeper      = "keeper"
 	AccessTokenAuthError = "HTTP response with status code 403"
 	//nolint: gosec
-	SecretsAuthError = "Received a '403' response"
+	SecretsAuthError                     = "Received a '403' response"
+	EnvEdgeXUseCommonAppServiceSecretKey = "EDGEX_USE_COMMON_APP_SERVICE_SECRET_KEY"
 )
```

Comment:
move to here
https://github.com/edgexfoundry/go-mod-bootstrap/blob/82d650d4162be9cc68f6bd1852cb96794abd15f2/bootstrap/environment/variables.go#L50